frontend/src/container/NotificationWall.jsx

```diff
@@ -159,6 +160,19 @@ export class NotificationWall extends React.Component {
     return { icon, text, url }
   }
 
+  handleClickMarkAllAsRead = async () => {
+    const { props } = this
+
+    const fetchAllPutNotificationAsRead = await props.dispatch(putAllNotificationAsRead(props.user.userId))
+    switch (fetchAllPutNotificationAsRead.status) {
+      case 204:
+        props.notificationPage.list.forEach(notification => props.dispatch(readNotification(notification.id)))
```

Comment:
I think it would be more efficient if we add a `readAllNotification` action in redux which will map the `list` without any condition (`eg .map(no => ({ ...no, read: true }))`) and `notificationNotReadCount` set to 0, it will avoid useless loop

Reply:
Changed